content/en/docs/operations/talos/configuration/air-gapped.md

```diff
@@ -100,3 +100,60 @@ Make apply for every node
 ```bash
 talm apply -f nodes/srv1
 ```
+
+### Make sure you have cozystack version 0.31.2 or higher
```

Comment:
No such version yet. Are we going to release the air-gap features in 0.31.2 or 0.32.0-alpha?